lib/main.dart

```diff
@@ -38,51 +38,48 @@ class MyApp extends HookConsumerWidget {
       animationNotifier.setController(animationController);
     });
 
-    return MaterialApp(
-        debugShowCheckedModeBanner: false,
-        home: WillPopScope(
-            onWillPop: () async {
-              final topBarCallBack = ref.watch(topBarCallBackProvider);
-              if (QR.currentPath.split('/').length <= 2) {
-                final animation = ref.watch(animationProvider);
-                if (animation != null) {
-                  final controllerNotifier =
-                      ref.watch(swipeControllerProvider(animation).notifier);
-                  controllerNotifier.toggle();
-                  topBarCallBack.onMenu?.call();
-                }
-                return false;
-              }
-              topBarCallBack.onBack?.call();
-              return true;
-            },
-            child: MaterialApp.router(
-              debugShowCheckedModeBanner: false,
-              title: 'MyECL',
-              scrollBehavior: MyCustomScrollBehavior(),
-              supportedLocales: const [Locale('en', 'US'), Locale('fr', 'FR')],
-              localizationsDelegates: const [
-                GlobalMaterialLocalizations.delegate,
-                GlobalWidgetsLocalizations.delegate,
-                GlobalCupertinoLocalizations.delegate,
-              ],
-              theme: ThemeData(
-                  primarySwatch: Colors.orange,
-                  textTheme:
-                      GoogleFonts.latoTextTheme(Theme.of(context).textTheme),
-                  brightness: Brightness.light),
-              routeInformationParser: const QRouteInformationParser(),
-              builder: (context, child) {
-                if (child == null) {
-                  return const SizedBox();
-                }
-                return AppTemplate(
-                  child: child,
-                );
-              },
-              routerDelegate:
-                  QRouterDelegate(appRouter.routes, initPath: AppRouter.root),
-            )));
+    return WillPopScope(
```

Comment:
You can't remove the MaterialApp, because I think that it will break phone version of the app. Please check it